Extension/src/LanguageServer/client.ts

```diff
@@ -581,7 +582,7 @@ interface FilesEncodingChanged {
 export interface CopilotCompletionContextResult {
     requestId: number;
     isResultMissing: boolean;
-    snippets: SnippetEntry[];
+    snippets: SupportedContextItem[];
```

Comment:
I'm seeing the actual data still be a SnippetEntry. Does that matter? i.e. the SupportedContextItem doesn't seem to have entries like startLine.

```
export interface SnippetEntry {
    uri: string;
    value: string;
    startLine: number;
    endLine: number;
    importance: number;
}
```